cmake/buildinfo/buildinfo.cmake

```diff
@@ -1,18 +1,28 @@
 cmake_host_system_information(RESULT CPU_NAME QUERY PROCESSOR_NAME)
 cmake_host_system_information(RESULT CPU_DESC QUERY PROCESSOR_DESCRIPTION)
 
+if (NOT DEFINED NBL_ROOT_PATH_BINARY)
+	message(FATAL_ERROR "NBL_ROOT_PATH_BINARY is not defined or it's empty")
+endif
+
 add_custom_command(
 	OUTPUT cmake_info.json
 	COMMAND ${CMAKE_COMMAND} -E capabilities >> cmake_info.json
 	COMMENT "Generating ${NBL_ROOT_PATH_BINARY}/cmake_info.json"
 )
-
-set(VKSDK_INFO_COMMAND "$ENV{VULKAN_SDK}/bin/vulkaninfoSDK")
-add_custom_command(
-	OUTPUT vulkan_info.json
-	COMMAND ${VKSDK_INFO_COMMAND} -j -o ${NBL_ROOT_PATH_BINARY}/vulkan_info.json
-	COMMENT "Generating ${NBL_ROOT_PATH_BINARY}/vulkan_info.json"
-)
+if (Vulkan_FOUND)
+	set(VKSDK_INFO_COMMAND "$ENV{VULKAN_SDK}/bin/vulkaninfoSDK")
+	add_custom_command(
+		OUTPUT vulkan_info.json
+		COMMAND ${VKSDK_INFO_COMMAND} -j -o ${NBL_ROOT_PATH_BINARY}/vulkan_info.json
+		COMMENT "Generating ${NBL_ROOT_PATH_BINARY}/vulkan_info.json"
+	)
+else()
+	add_custom_command(
+		OUTPUT vulkan_driver_disabled.txt
+		COMMAND ${CMAKE_COMMAND} -E touch ${NBL_ROOT_PATH_BINARY}/vulkan_driver_disabled.txt
+		COMMENT "Generating ${NBL_ROOT_PATH_BINARY}/vulkan_driver_disabled.txt"
```

Comment:
https://github.com/Devsh-Graphics-Programming/Nabla/pull/859#discussion_r2019926984 + https://github.com/Devsh-Graphics-Programming/Nabla/commit/7e00fd639581416363f5c970eff568b229f79fa2#r154593064

Reply:
typo + how did you test it when it never configured?

Reply:
done by me